infra/config-store/src/main/proto/project_config.proto

```diff
@@ -96,6 +96,8 @@ message ProtoFingerprintConfiguration {
         optional ProtoVero1Configuration vero_1 = 5;
         optional ProtoAllowedAgeRange allowed_age_range = 6;
         optional float verification_match_threshold = 7;
+
```

Comment:
Extra line

Reply:
Blank space is prepared for the addition of extra config that is already in main branch. Same reason it skips a number as well.